resources/cygnus-translators/README.md

```diff
@@ -132,7 +133,59 @@ $ hadoop fs -cat /user/johndoe/dataset/data.txt
 
 [Top](#top)
 
-##<a name="section5"></a>Reporting issues and contact information
+##<a name="section5"></a>`cygnus_translator_pre0.10.0_to_0.10.0_mysql.sh`
+This scripts adds certain fields not available in previous versions to 0.10.0 to the MySQL tables potentially containing historical context data. Specifically:
+
+* Tables written in `row` format will have a new fields named `fiwareservicePath`.
+* Tables written in `column` format will have new fields named `fiwareservicePath`, `entityId` and `entityType`.
+* 
+Usage:
+
+```
+$ ./cygnus_translator_pre0.10.0_to_0.10.0_mysql.sh 
+Usage: cygnus_translator_pre0.10.0_to_0.10.0_mysql.sh user password database tableFormat backup
+where user        : a valid user in the MySQL server granted to modify the below database
+      password    : password for the above user
+      database    : a valid database name
+      table_format: a value within row|column
```

Comment:
Suggestion: "a value within row|colum" -> "either 'row' or 'column'.

Similar style suggestion also for "backup" field.